src/core_plugins/kibana/public/home/components/tutorial/instruction_set.js

```diff
@@ -60,6 +65,61 @@ export class InstructionSet extends React.Component {
     ));
   }
 
+  renderStatusCheckMsg(msg, color) {
+    return (
+      <EuiCallOut
+        title={msg}
+        color={color}
+      />
+    );
+  }
+
+  renderStatusCheck() {
+    let statusMsg;
+    if (this.props.statusCheckState === 'complete') {
+      const msg = this.props.statusCheckConfig.success ? this.props.statusCheckConfig.success : 'Success';
+      statusMsg = this.renderStatusCheckMsg(msg, 'success');
+    } else if (this.props.hasStatusCheckFailed) {
+      const msg = this.props.statusCheckConfig.error ? this.props.statusCheckConfig.error : 'No data found';
+      statusMsg = this.renderStatusCheckMsg(msg, 'warning');
+    }
+
+    const checkStausStep = (
+      <Fragment>
+        <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+          <EuiFlexItem>
+            <EuiText>
+              <p>
+                {this.props.statusCheckConfig.text}
+              </p>
+            </EuiText>
+          </EuiFlexItem>
+
+          <EuiFlexItem
+            grow={false}
+          >
+            <EuiButton
+              onClick={() => {this.props.onStatusCheck();}}
```

Comment:
nit: i think the spacing should be `onClick={() => { this.props.onStatusCheck(); }}`  

or actually I think you can just do:

`onClick={this.props.onStatusCheck}`